### PR TITLE
Add explainer document.

### PR DESCRIPTION
**Pull request by guidou**
@dontcallmedom can you review this first version of an explainer document?

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+For example:
+
+```js
+var audioElement = new Audio('sound.ogg');
+audioElement.play();
+navigator.mediaDevices.enumerateDevices().then(devices => {
+  var lastOutputDeviceId = '';
+  for (i in devices) {
+    if (devices[i].kind == 'audiooutput') {
+      lastOutputDeviceId = devices[i].id;
```
Comment (dontcallmedom) [inline explainer.md]: s/devices[i].id/devices[i].deviceId/

File: explainer.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+  audioElement.setSinkId(lastOutputDeviceId).then(() => {
+    console.log('Audio output device set to ' + lastOutputDeviceId.sinkId);
```
Comment (dontcallmedom) [inline explainer.md]: s/lastOutputDeviceId.sinkId/lastOutputDeviceId/

File: explainer.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  }).catch(err => {
+    console.log('setSinkId() failed', err);
+  );
+}).catch(err => {
+  console.log('enumerateDevices() failed :(', err);
+}
```
Comment (dontcallmedom) [inline explainer.md]: missing closing parenthesis

File: explainer.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  for (i in devices) {
+    if (devices[i].kind == 'audiooutput') {
+      lastOutputDeviceId = devices[i].id;
+    }
+  }
+  audioElement.setSinkId(lastOutputDeviceId).then(() => {
+    console.log('Audio output device set to ' + lastOutputDeviceId.sinkId);
+  }).catch(err => {
+    console.log('setSinkId() failed', err);
+  );
```
Comment (dontcallmedom) [inline explainer.md]: missing closing curly brace

File: explainer.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+HTML media elements. The setSinkId() method allows changing the audio output
+device on which the element renders audio, and the sinkId attribute is a string
+that contains the ID of the device where the element is currently rendering
+audio.
+
+By default, the sinkId field is the empty string, which means that audio will be
+rendered on the default audio output device in the system.
+
+The set of valid device IDs that can be used as argument to setSinkId() can be
+obtained from a call to [enumerateDevices](https://w3c.github.io/mediacapture-main/#dom-mediadevices-enumeratedevices).
```
Comment (dontcallmedom) [inline explainer.md]: Maybe mention that permissions for non-default audio output devices are linked to `getUserMedia` permissions

Comment (guidou) [reply explainer.md]: done

File: explainer.md (hunk 1 above)
Comment (guidou) [reply explainer.md]: done

File: explainer.md (hunk 2 above)
Comment (guidou) [reply explainer.md]: done

File: explainer.md (hunk 4 above)
Comment (guidou) [reply explainer.md]: done

File: explainer.md (hunk 3 above)
Comment (guidou) [reply explainer.md]: done

**Pull request merged**